presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -203,9 +203,12 @@ Property Name                                Description
 
 ``hive.s3.aws-secret-key``                   Default AWS secret key to use.
 
-``hive.s3.endpoint``                         The S3 storage endpoint server. This can be used to
-                                             connect to an S3-compatible storage system instead
-                                             of AWS.
+``hive.s3.endpoint``                         The S3 storage endpoint server. Set this to connect to an
+                                             S3-compatible storage system instead of AWS. You can also
+                                             set this to AWS region-specific endpoint such as
+                                             ``http[s]://<bucket>.s3-<AWS-region>.amazonaws.com`` if
```

Comment:
Suggest: When using V4 signatures, it is recommended to set this to the AWS region-specific endpoint (e.g. ``http[s]://<bucket>.s3-<AWS-region>.amazonaws.com``).

The issue also came up when connecting to an S3 bucket in the same region as the EC2 instance where Presto is running, so we shouldn't mention the S3 bucket being in a different region.